ovirt_imageio/client/_ui.py

```diff
@@ -52,8 +54,24 @@ def draw(self, elapsed, size, value, transferred, phase=None, last=False):
         self._write_output(line, end)
 
 
+class JsonFormat(OutputFormat):
+
+    def draw(self, elapsed, size, value, transferred, phase=None, last=False):
+        line_dict = {
```

Comment:
Do we really need to call this line_dict? it is obvious that this is a dict
since we have a literal dict and the entire function is 10 lines.

I would call this `progress` since this function draws a json progress.